content/en/monitors/incident_management/_index.md

```diff
@@ -0,0 +1,185 @@
+
+Read more about using the Datadog Slack App [here][8]. 
+
+{{< img src="monitors/incidents/from-slack.png" alt="Create in incident from Slack" style="width:60%;">}}
+
+If the user declaring the incident is a part of your Datadog account, then that user becomes the Incident Commander (IC) by default. If the person declaring an incident is not part of your Datadog account, then the IC is assigned to a generic `Slack app user`. The IC can be changed on the [incidents page][1] in the Datadog app.
+
+Once you declare an incident from Slack, it generates an incident channel.
+
+For more information about the Datadog Slack integration, check out [the docs][3].
+
+### Describing the incident
+
+No matter where you create an incident, it's important to describe it as thoroughly as possible to share the information with other people involved in your company's incident management process.
+
+When you create an incident, an incident modal comes up. This modal has several core elements:
+
+**Severity Level**: Denotes the severity of your incident, from SEV-1 (most severe) to SEV-5 (least severe). If your incident is under initial investigation, and you do not know the severity yet, select UNKNOWN.
+
+* SEV-1: Critical impact
+* SEV-2: High impact
+* SEV-3: Moderate impact
+* SEV-4: Low impact
+* SEV-5: Minor issue
+* UNKNOWN: Initial investigation
+
+**Note**: You can customize the description of each severity level to fit the requirements of your organization.
+
+**Title**: Give your incident a descriptive title.
+
+**Signals**: The reason(s) you are declaring the incident. These can be graphs, logs, or other key visuals.
+
+**Incident commander**: This person is assigned as the leader of the incident investigation.
+
+**Additional notifications**: Notify other teams or people.
+
+Click on “Declare Incident” to finish creating your incident.
+
+### Updating the incident and the incident timeline
+
+An incident’s status can be updated directly in the [overview page][1] of the incident or from Slack within the dedicated incident channel. To update an incident from Slack, use this slash command to pull up the update modal: `/datadog incident update`
+
+You can also update the impact section to specify if there was customer impact, the incident timeline, and whether or not it’s still active. This section also requires a description of the scope of impact to be completed.
+
+In the incident header, you can see the incident's state, severity, timestamp, impact, and duration, as well as who has responded to the incident. You can also notify responders of updates. There are quick links to chat channels, video conferencing, and attached postmortem (if one has been added).
```

Comment:
There are quick links to add and access chat channels (if not using the Datadog Slack App), video conferencing,